apps/dapp-console-api/src/routes/faucet/FaucetRoute.spec.ts

```diff
@@ -0,0 +1,221 @@
+      ],
+      account: faucetAdminAccount,
+      chain: walletClient.chain,
+    })
+
+    const route = new FaucetRoute(trpc, [
+      new Faucet({
+        chainId: 11155111,
+        redisCache: redisCache,
+        faucetAddress: faucetAddress,
+        displayName: 'OP Sepolia',
+        onChainDripAmount: parseEther('1.0'),
+        offChainDripAmount: parseEther('0.05'),
+        onChainAuthModuleAddress: onChainFamAddress,
+        offChainAuthModuleAddress: offChainFamAddress,
+        blockExplorerUrl:
+          'https://sepolia.infura.io/v3/4a5209fff1e944f6b773150e48c06862',
+        isL1Faucet: false,
+        l1BridgeAddress: '0xfd0Bf71F60660E2f608ed56e1659C450eB113120',
+      }),
+    ]).handler
+    caller = createSignedInCaller(route, session)
+  })
+
+  afterEach(async () => {
+    vi.restoreAllMocks()
+  })
+
+  describe('faucetsInfo', () => {
+    it('returns the expected response', async () => {
+      vi.spyOn(redisCache.redisClient, 'get').mockImplementation(() => {
+        return Promise.resolve(
+          JSON.stringify({ type: 'bigint', value: `${parseEther('4200')}` }),
+        )
+      })
+
+      const faucetsInfo = await caller.faucetsInfo()
+
+      expect(faucetsInfo).toEqual([
+        {
+          chainId: 11155111,
+          displayName: 'OP Sepolia',
+          isAvailable: true,
+          onChainDripAmount: parseEther('1.0'),
+          offChainDripAmount: parseEther('0.05'),
+        },
+      ])
+    })
+
+    it('returns isAvailable as false when balance is less than 1 eth', async () => {
+      vi.spyOn(redisCache.redisClient, 'get').mockImplementation(() => {
+        return Promise.resolve(
+          JSON.stringify({ type: 'bigint', value: `${parseEther('0')}` }),
+        )
+      })
+
+      const faucetsInfo = await caller.faucetsInfo()
+
+      expect(faucetsInfo[0].isAvailable).toBeFalsy()
+    })
+
+    it('returns isAvailable as true when balance is greater than or equal to 1 eth', async () => {
+      vi.spyOn(redisCache.redisClient, 'get').mockImplementation(() => {
+        return Promise.resolve(
+          JSON.stringify({ type: 'bigint', value: `${parseEther('1')}` }),
+        )
+      })
+
+      const faucetsInfo = await caller.faucetsInfo()
+
+      expect(faucetsInfo[0].isAvailable).toBeTruthy()
+    })
+
+    it('returns isAvailable as true when balance is returned as null', async () => {
+      vi.spyOn(redisCache.redisClient, 'get').mockImplementation(() => {
+        return Promise.resolve(null)
+      })
+
+      const faucetsInfo = await caller.faucetsInfo()
+
+      expect(faucetsInfo[0].isAvailable).toBeTruthy()
+    })
+
+    it('returns isAvailable as true when fetching the balance returns an error', async () => {
+      vi.spyOn(redisCache.redisClient, 'get').mockRejectedValue(
+        new Error('Redis is offline'),
+      )
+
+      const faucetsInfo = await caller.faucetsInfo()
+
+      expect(faucetsInfo[0].isAvailable).toBeTruthy()
+    })
+  })
+})
```

Comment:
can the test suites for `nextDrips`, `onChainClaims`, and `offChainClaims` be copied over as well?